packages/components/src/components/primitives/Divider/Divider.test.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { renderWithProviders } from 'tests/renderWithProviders';
+import TestRendered from 'react-test-renderer';
+import { Divider } from './Divider';
+
+describe('Divider', () => {
+  it('Should render', () => {
+    const { getByTestId } = renderWithProviders(
+      <Divider data-testid="divider-id" />
+    );
+
+    getByTestId('divider-id');
```

Comment:
use `screen.getByTestId` instead, see https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#not-using-screen

Reply:
done!.